core/celo_genesis.go

```diff
@@ -109,31 +110,29 @@ func celoGenesisAccounts() map[common.Address]GenesisAccount {
 			Code:    feeCurrencyWhitelistBytecode,
 			Balance: big.NewInt(0),
 			Storage: map[common.Hash]common.Hash{
-				common.HexToHash("0x1"):                               common.HexToHash("0x1"),    // array length 1
-				crypto.Keccak256Hash(common.HexToHash("0x1").Bytes()): common.HexToHash("0xce16"), // FeeCurrency
+				common.HexToHash("0x1"):                               common.HexToHash("0x1"),                // array length 1
+				crypto.Keccak256Hash(common.HexToHash("0x1").Bytes()): common.BytesToHash(FeeCurrencyAddr[:]), // FeeCurrency
```

Comment:
To get the byte array, otherwise we run into:
```
cannot use FeeCurrencyAddr (variable of type common.Address) as []byte value in argument to common.BytesToHash
```

We can also use `FeeCurrencyAddr.Bytes()` if that makes it clearer.

Reply:
Yep, I would use that instead.